regulations/static/regulations/js/source/collections/comment-collection.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.LocalStorage = require('backbone.localstorage');
+
+var CommentModel = require('../models/comment-model');
+
+var CommentCollection = Backbone.Collection.extend({
+  model: CommentModel,
+  localStorage: new Backbone.LocalStorage('comment:' + window.DOC_NUMBER)
```

Comment:
I don't think we should encode the assumption that a user will only ever want to see comments for a specific proposal if we don't have to. Adding in the requirement for global state, this seems like a worse recipe. Consider we could easily create a function (or sub-type this collection) which filters to a specific doc number.